pennylane/devices/experimental/default_qubit_2.py

```diff
@@ -135,13 +136,15 @@ def f(x):
 
     """
 
+    pennylane_requires = __version__
+
```

Comment:
We could instead just avoid the check in `qml.device` for if the property isn't present.